modules/gmake/src/sln.lua

```diff
@@ -0,0 +1,251 @@
+---
+-- GNU Makefile solution exporter.
```

Comment:
I realize you were just lifting from the existing VS exporter, but for makefiles these should be called "workspace" and "wks" rather than "solution" and "sln", which are VS specific terms. Same with the unit test names, etc.